algorithms/ar-bsrh/java/LinearSearch.java

```diff
@@ -0,0 +1,36 @@
+import java.util.*;
+
+public class LinearSearch {
+    public static void main (String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int n, x, q;
```

Comment:
Codacy found an issue: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6278826)